### ISSUE TITLE
Add alt text to images

### ISSUE THREAD

**Issue by detnos** [daf-cgoc-2025/nationalcgoc.github.io#10]
use the below file to update alt text on images that do not have alt text:

[DAF CGOC missing alt text file](https://docs.google.com/spreadsheets/d/1xo1Pc6WYay1ZdiEXZWaQnulwUb7NrhNNE2bUFn-AMEM/edit?pli=1&gid=1#gid=1)

**Comment by detnos** [comment]
first time doing this with google docs. Let me know if there are any hiccups.

**Comment by detnos** [comment]
Completed this task on my own kaci is not contributing.
